app/helpers/answer_helper.rb

```diff
@@ -0,0 +1,79 @@
+#E1924 Spring 2019 Addition
+
+module AnswerHelper
+
+  # Delete responses for given questionnaire
+  def self.delete_existing_responses(question_ids, questionnaire_id)
+    response_ids=[]
+    question_ids.each do |question|
```

Comment:
instead of iterating over question, iterate over id. 
upon first look, I think we're iterating over a list of questions
You do it correctly below and it's way more readable

Reply:
Not sure where you are referring to. We iterate through question here to get the list of response_ids which we iterate through later

Reply:
you're iterating over the ids, not the question object. 
change 
```ruby
question_ids.each do |question|
```
to 
```ruby
question_ids.each do |question_id|
```